### ISSUE TITLE
design "SF-SAC: Audit Information" page for implementation based on sprint 6 research insights

### ISSUE THREAD

**Issue by austinhernandez** [GSA-TTS/FAC#305]
Breakout issue of #274 

**Background**
[Sprint 6 research -  observations/refinements by page](https://docs.google.com/document/d/1Y8g7HEd6eysBQjg3x_-K3nzk9M9xBCp5w8eZXql_8xQ/edit#heading=h.ycduc4d91ltn)

**Observation(s)**
- All users understood the questions being asked here and how to navigate the form.
- One user asked about the dollar threshold & Type B programs, and if that was still necessary. Another said it was when asked. 
- A few users pointed out that the federal agencies list is incomplete and the format of checkboxes, while more intuitive than it was in the current version, would be cumbersome to use for the full list of agencies that would have to go here. (note from @MelissaBraxton "We should double check this. I think the users that asked about that may be misunderstanding what the list actually is and why it appears on this page. We should get to the bottom of the underlying need here and then adjust the recommendation.")
- Another user inquired why is this list here at all, though was short on details.

**MVP recommendation(s)**
- The agencies list will need to be updated to accommodate many more options. Checkboxes were appreciated but there may need to be a different way to display these. ((note from @MelissaBraxton "May need to revise based on above.")

**Tasks**
- [x] Review sprint 6 research analysis
- [x] Create checklists for refinement needed to each page
- [x] Update #274 with the checklist of refinements needed
- [ ] Make refinements
- [ ] Peer review

**Definition of done**
**Acceptance Criteria (We'll know we're done when...)**
- [ ] Screens in Figma are ready to be handed off to front end
- [x]  All text has a contrast ratio of 4.5:1 with the background
- [ ]  Interaction are documented if they deviate from USWDS components
- [ ]  The needs of someone navigating the site with the use of a screenreader or keyboard has been considered and documented 
- [ ] There is an option for multilingual support
- [ ] There is supporting documentation for alt text, labels for links (anchor + target), error text in forms, and what assistive technologies should announce (if applicable)

**Comment by melchoyce** [comment]
Is this referencing this Figma page? https://www.figma.com/file/1bNncVt1TcFFDhQslqFC2o/FAC-Playground-(Copy)?node-id=2464%3A10742?

**Comment by melchoyce** [comment]
Thanks for this context — let me try a version where part of the list is visible by default. 👍

**Comment by sreuber** [comment]
@austinhernandez Yes, I simply lifted what the current site was doing when I originally designed this page way back when, so change what's needed.

**Comment by melchoyce** [comment]
Updated the list:

![image](https://user-images.githubusercontent.com/2846578/178845393-0a4685f8-afe1-47ae-a9fd-795a8ea37c71.png)

Going to write up implementation notes next.

**Comment by melchoyce** [comment]
@petermarks-gsa Implementation notes written: https://www.figma.com/file/1bNncVt1TcFFDhQslqFC2o/FAC-Playground?node-id=3623%3A17578
